internal/testutils/rlimit.go

```diff
@@ -0,0 +1,20 @@
+package testutils
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/cilium/ebpf/internal/unix"
+)
+
+func init() {
```

Comment:
@ti-mo this changed since your last review. I generally don't like to use `init` but it seems appropriate here. What do you think?